rust/blockstore/src/arrow/ordered_blockfile_writer.rs

```diff
@@ -133,6 +144,25 @@ impl ArrowOrderedBlockfileWriter {
 
         Self::complete_current_delta::<K, V>(&mut inner);
 
+        let num_block_cache_hits = self
+            .num_block_cache_hits
+            .load(std::sync::atomic::Ordering::Relaxed);
```

Comment:
any particular reason that the ordering is Relaxed here and not SeqCst?

Reply:
it's what popped up in my autocomplete first
it doesn't matter here since there should be no competing writes at this point